Assets/Scripts/Enemy/EnemyTest.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyTest : EnemyController
+{
+    //[SerializeField] protected GameObject playerObject;
+    protected int EnemyTestHealth = 20;
```

Comment:
I imagine this is not serialized because we intend to change this value via the spawner? Even so, it should private with a getter and setter

Reply:
This field should also be declared in `CharController` since all characters in the game will have a health value

Reply:
Changed to "[SerializeField] private". Health is declared again in the enemy scripts, to have different healths for different enemies